fala/apps/adviser/forms.py

```diff
@@ -64,6 +64,24 @@ def clean(self):
         data = self.cleaned_data
         if not data.get("postcode") and not data.get("name"):
             raise forms.ValidationError(_("Enter a postcode or an organisation name"))
+        else:
+            msg1 = "This service does not cover "
+            msg2 = "Try a postcode, town or city in England or Wales."
+            x = "England or Wales. "
```

Comment:
Can we name this something more descriptive like territory or region

Reply:
The wording was provided by Josh.  Or were you thinking to replace "England or Wales"?